### DISCUSSION TITLE
Quasar Theme Builder as Chrome/FF extension

### DISCUSSION THREAD

**Discussion by glatocha** [quasarframework/quasar#11674] [Ideas / Proposals]
Hi,
it would be awesome to have a Theme Builder as a devTools extension in Chrome/FF.
That way one could play around the colors in own app already.

cheers
g.

P.S. Great job guys!!

**Comment by hawkeye64** [comment]
@glatocha Which Theme Builder are you referring to? This one? https://quasar.dev/style/theme-builder#introduction
Sounds like fun for someone with time on their hands.

**Reply by glatocha** [reply]
Yeah, that one. I'll read a little but as everyone have some other projects on hand so far. As much much to learn
